BackEndApp/authv1/views.py

```diff
@@ -93,3 +96,96 @@ def logout(request):
         status = 500
 
     return JsonResponse({"message": message}, status=status)
+
+
+@api_view(["POST"])
+def forgot_password(request):
+    try:
+        username = request.data.get("username")
+        user = User(username=username, password=None)
+        this_user = user.find()
+
+        if user is None:
+            message = "User does not exist"
```

Comment:
Not a good message in terms of security .. Can be something like "Invalid Credentials". @ADI10HERO

Reply:
Agreeed 💯

Reply:
Will fix it.